src/test/resources/application-test.yml

```
@@ -0,0 +1,47 @@
+spring:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)